src/lib.rs

```diff
@@ -100,6 +100,9 @@ pub enum Error {
     SharingNotMaximal,
     /// Miniscript Error
     MiniscriptError(miniscript::Error),
+    /// Policy-related error
+    #[cfg(feature = "elements")]
+    Policy(policy::Error),
```

Comment:
In 9e6381d84fd06d5d25d043597c4a1e83253d00b3:

You cannot feature-gate enum variants unless the enum is marked `#[non_exhaustive]`. Otherwise enabling the feature will break compilation for downstream users.

Reply:
Having a separate policy error would be useful. Maybe I can put it in a non-feature-gated module?

Reply:
I think it's fine to just mark the error enum as `#[non_exhaustive]`.

Reply:
One rarely matches over all error variants, so this is probably fine